demos/http3/ossl-nghttp3-demo-server.c

```diff
@@ -319,6 +319,10 @@ static int nothing_from_ssl_ids(nghttp3_conn *h3conn, struct h3ssl *h3ssl)
             printf("revent %llu (%d) on %llu\n",
                    (unsigned long long)item->revents, SSL_POLL_EVENT_W,
                    (unsigned long long)ssl_ids[i].id);
+        } else if (item->revents == SSL_POLL_EVENT_ER) {
+            printf("revent %llu (%d) on %llu\n",
+                   (unsigned long long)item->revents, SSL_POLL_EVENT_ER,
+                   (unsigned long long)ssl_ids[i].id);
```

Comment:
Just a nitpick, but this could be one `else if` if I read it right.

```c
} else if (item->revents == SSL_POLL_EVENT_ER || item->revents == SSL_POLL_EVENT_W) {
            printf("revent %llu (%d) on %llu\n",
                   (unsigned long long)item->revents, item->revents,
                   (unsigned long long)ssl_ids[i].id);
}
```

(I'm still getting notifications about the changes so that's why I appear from time to time :smile:)

Reply:
Alright then.